1-index.html

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Jamulus ‒ Play music online. With friends. For free."
+title: "Jamulus ‒ Jam online. With friends. For free."
```

Comment:
From an SEO point of view, I agree that "jam" is probably going to be a more realistic term for us to target than "play". It's just a pity that perhaps only a small subset of people actually use Jamulus to jam with.